### DISCUSSION TITLE
运动轨迹放大后发现轨迹不够连续，都是折线，能加密吗？

### DISCUSSION THREAD

**Discussion by efish2002** [yihong0618/running_page#433] [General]
运动轨迹放大后发现间隔较大，程序中这个是固定值吗？对从app中导入的轨迹重采样了吗？间隔能修改吗？

**Comment by yihong0618** [comment]
是固定值，得在上游 gpxpy 里修改

**Reply by yihong0618** [reply]
可以加密，搜索 issue 里面有方法

**Reply by efish2002** [reply]
没有在issue里找到解决办法，还请指教

**Reply by yihong0618** [reply]
https://github.com/yihong0618/running_page/issues/353 
https://github.com/yihong0618/running_page/issues/226

用这两种方式。（第二种需要是 strava

**Reply by efish2002** [reply]
是我没表达清楚，我说的加密是把运动轨迹的取点间隔加密，我发现gpx文件的点间隔就比较大，说明数据从Garmin中保存成gpx文件的时候就重采样了。

**Reply by yihong0618** [reply]
这个我研究一下，有可能可以

**Comment by efish2002** [comment]
注释掉track.py 中的gpx.simplify()，轨迹就好多了。

**Reply by yihong0618** [reply]
@efish2002 谢谢，我研究下是不是可以改成默认的。
